src/analysis/retail/rogue/subtlety/modules/spells/SymbolsOfDeath.tsx

```diff
@@ -0,0 +1,191 @@
+      this.onCast,
+    );
+  }
+
+  get guideSubsection(): JSX.Element {
+    const explanation = (
+      <p>
+        <strong>
+          <SpellLink spell={SPELLS.SYMBOLS_OF_DEATH} />
+        </strong>{' '}
+        is a key cooldown for burst windows. It should be used as frequently as possible, ideally
+        aligning with <SpellLink spell={TALENTS.SECRET_TECHNIQUE_TALENT} /> for maximum damage
+        output.
+      </p>
+    );
+
+    const goodCasts = this.cooldownUses.filter(
+      (it) =>
+        it.performance === QualitativePerformance.Good ||
+        it.performance === QualitativePerformance.Perfect,
+    ).length;
+    const totalCasts = this.cooldownUses.length;
+
+    return (
+      <HideGoodCastsSpellUsageSubSection
+        hideGoodCasts={false}
+        explanation={explanation}
+        uses={this.cooldownUses}
+        castBreakdownSmallText={<> - Red indicates a wasted Symbols of Death.</>}
+        onPerformanceBoxClick={logSpellUseEvent}
+        abovePerformanceDetails={
+          <div style={{ marginBottom: 10 }}>
+            <CastPerformanceSummary
+              spell={SPELLS.SYMBOLS_OF_DEATH}
+              casts={goodCasts}
+              performance={
+                this.cooldownUses.length > 0
+                  ? this.cooldownUses[0].performance
+                  : QualitativePerformance.Fail
+              }
+              totalCasts={totalCasts}
+            />
+          </div>
+        }
+        noCastsTexts={{
+          noCastsOverride: 'No Symbols of Death casts detected! This is a major mistake.',
+        }}
+      />
+    );
+  }
+
+  private onCast(event: CastEvent) {
+    const energyAtCast = this.energyTracker.current;
+    const hasSecretTechnique =
+      this.selectedCombatant.hasTalent(TALENTS.SECRET_TECHNIQUE_TALENT) &&
+      this.selectedCombatant.hasBuff(TALENTS.SECRET_TECHNIQUE_TALENT.id, event.timestamp);
+
+    this.cooldownUses.push(
+      createSpellUse({ event }, [
+        this.energyPerformance(event, energyAtCast, hasSecretTechnique),
+        this.secretTechniquePerformance(event, hasSecretTechnique),
+      ]),
+    );
+  }
+
+  private energyPerformance(
+    event: CastEvent,
+    energyAtCast: number,
+    hasSecretTechnique: boolean,
+  ): ChecklistUsageInfo | undefined {
+    const energyGained = 40;
+    const wastedEnergy = Math.max(0, energyAtCast + energyGained - 100);
+    const isEnergyWasted = wastedEnergy > 0;
+
+    let performance: QualitativePerformance;
+    let summary: JSX.Element;
+    let details: JSX.Element;
+
+    if (energyAtCast === 100 && !hasSecretTechnique) {
+      performance = QualitativePerformance.Fail;
+      summary = <div>Major Energy Waste</div>;
+      details = (
+        <div>
+          You used <SpellLink spell={SPELLS.SYMBOLS_OF_DEATH} /> at **{energyAtCast}** energy
+          without **Secret Technique**. This means you **wasted {wastedEnergy} energy**. Try using
+          abilities before activating Symbols of Death!
```

Comment:
note that markdown formatting is not supported here. you probably want to use `<strong>` instead to bold these properly:

```suggestion
          You used <SpellLink spell={SPELLS.SYMBOLS_OF_DEATH} /> at <strong>{energyAtCast}</strong> energy
          without <strong>Secret Technique</strong>. This means you <strong>wasted {wastedEnergy} energy</strong>. Try using
          abilities before activating Symbols of Death!
```